README.md

```diff
@@ -702,7 +702,7 @@ buildctl \
 
 ### Load balancing
 
-`buildctl build` can be called against randomly load balanced the `buildkitd` daemon.
+`buildctl build` can be called against randomly load balanced `buildkitd` daemon.
```

Comment:
I guess it should be daemons, in plural, here

Reply:
Added s